GliaWidgets/Glia.swift

```diff
@@ -16,7 +16,7 @@ public enum GliaEvent {
     case maximized
 }
 
-public protocol SceneProvider: class {
+public protocol SceneProvider: AnyObject {
```

Comment:
Good, I was going to mention this at some point cause I saw the warning. Thanks for fixing it.

Reply:
> I don't see the operator is typing indicator on my device. I type and nothing happens on the device until I press send, where the device receives the phone. Also, very importantly, the operator does not see that the visitor is typing.

Indicating that the operator is typing is out of the scope of this PR. This PR only adds a raindrop-like indicator to the chat to indicate if any new messages have appeared while the visitor was scrolling through the chat (i.e. could not see them right away) - [[MUIC-279]](https://salemove.atlassian.net/jira/software/projects/MUIC/boards/132?label=iOS&selectedIssue=MUIC-279)